src/Plugins/Reminder.php

```diff
@@ -10,28 +10,28 @@ class Reminder extends NuntiusPluginAbstract {
   protected $category = 'Reminders';
 
   public $formats = [
-    '/remind me next time I logging in to (.*)/' => [
+    '/remind me next time I log into (.*)/' => [
```

Comment:
"Remind me next time I log in to..."

http://grammarist.com/spelling/log-in-login/